framework/components/ecl-forms/ecl-forms-fieldsets/_fieldsets.scss

```diff
@@ -0,0 +1,10 @@
+/**
+ * Form Fieldset
+ * @define fieldset
+ */
+
+.ecl-fieldset {
```

Comment:
60px spacing between last item of "Personal information" and "Group 2" on the image.